### PR TITLE
feat: add region sync, #136

### PR DESCRIPTION
**Pull request by sadams**
Adds utilities for doing region syncing. 
Relates to 5app/deployment#136

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,8 +40,41 @@
...
+		"@semantic-release/git": "^9.0.0",
+		"eslint": "^7.9.0",
+		"eslint-config-5app": "^0.12.3",
+		"eslint-config-prettier": "^6.11.0",
+		"eslint-plugin-prettier": "^3.1.4",
+		"husky": "^4.3.0",
+		"prettier": "^2.1.2",
+		"pretty-quick": "^3.0.2",
+		"semantic-release": "^17.1.2",
+		"@5app/logger": "^2.0.2",
```
Comment (MrSwitch) [inline package.json:60]: Should these not be in devDependencies?

File: package.json [hunk 2]
```diff
@@ -55,8 +40,41 @@
...
+		"pub-sub",
+		"pub",
+		"sub",
+		"region",
+		"aws",
+		"sync",
+		"queues",
+		"topics"
+	],
+	"author": "Sam Adams <adams.sam@gmail.com>"
```
Comment (MrSwitch) [inline package.json:79]: Are you after some unsolicited emails?

File: package.json [hunk 3]
```diff
@@ -55,8 +40,41 @@
...
 		}
-	}
+	},
+	"dependencies": {
+		"@commitlint/cli": "^11.0.0",
+		"@commitlint/config-conventional": "^11.0.0",
+		"@semantic-release/changelog": "^5.0.1",
+		"@semantic-release/git": "^9.0.0",
+		"eslint": "^7.9.0",
+		"eslint-config-5app": "^0.12.3",
```
Comment (MrSwitch) [inline package.json:53]: As it's a node project could you also add `eslint-plugin-node` and extend `.eslintrc` with `plugin:node/recommended`. Thats really good for catching weird things in package distributions.

As part of that you'll need a `files` section else it'll complain, something like 

```
files: ['src/','index.js'] 
```

File: package.json (hunk 1 above)
Comment (sadams) [reply package.json:60]: Good point.

Comment (sadams) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (sadams) [reply package.json]: changed

File: package.json (hunk 3 above)
Comment (sadams) [reply package.json:64]: done

**Review by MrSwitch — APPROVED** [review]

**Pull request merged**